src/SourceBuild/patches/runtime/0002-short-stack-support.patch

```diff
@@ -0,0 +1,41 @@
+--- a/eng/SourceBuild.props	2024-02-07 11:01:33.807337902 -0500
```

Comment:
Please add a Backport comment to the patch .  This is extremely helpful in the dependency flow PRs and tracking the removal of the patch.  Yes, I know the flow is already in works.

https://github.com/dotnet/source-build/blob/main/Documentation/patching-guidelines.md#patch-guidelines